App.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import AppNavigator from './src/screens';
 import {QueryClientProvider, QueryClient} from '@tanstack/react-query';
+import {SafeAreaView} from 'react-native';
+import Colors from './src/types/colors';
 
 const queryClient = new QueryClient();
 
 function App(): React.JSX.Element {
   return (
     <QueryClientProvider client={queryClient}>
-      <AppNavigator />
+      <SafeAreaView style={{backgroundColor: Colors.black, flex: 1}}>
```

Comment:
Don't use inline style as it will recreate on every render. If it's possible add this inside the navigator or in the screens